scripts/build_docker_containers

```diff
@@ -50,22 +28,31 @@ build_mil_docker_image()
   docker build $DOCKER_ARGS $DOCKER_BASE_PATH/$1 -t $MIL_DOCKER_TAG_ROOT:$1 --build-arg MIL_DOCKER_TAG_ROOT=${MIL_DOCKER_TAG_ROOT}
 }
 
-build_vrx_image()
```

Comment:
Why remove vrx?

Reply:
so we never used this container. All the containers we used were from the vrx-docker repo. This is in the spirit of getting rid of old things we dont use anymore while we still remember that we dont need them.